packages/core/src/utils/pushNotification.ts

```diff
@@ -0,0 +1,86 @@
+import axios from 'axios';
+import config from '../config';
+import { client as prismic } from '../utils/prismic';
+import { NotificationType } from '../interfaces/app/appNotification';
+import { models } from '../database';
+import { AppUserModel } from '../database/models/app/appUser';
+import localesConfig from '../utils/locale.json';
+
+export async function sendNotification(
+    users: AppUserModel[],
+    type: NotificationType,
+) {
+    // registry notification
+    await models.appNotification.bulkCreate(users.map(el => ({
+        userId: el.id,
+        type,
+        isWallet: true,
+        isWebApp: true,
```

Comment:
So far, all push notifications go to wallet and app, but likely won't be the case in the future. It's better if you provide options on the method.